lib/webtrit_adapter_client.ex

```diff
@@ -230,6 +230,28 @@ defmodule WebtritAdapterClient do
     request(client, options)
   end
 
+  @spec invoke_custom_public_method(Tesla.Client.t(), String.t(), String.t(), map()) :: result()
+  def invoke_custom_public_method(client, method_name, extra_params, data) do
+    options = [
+      method: :post,
+      url: "/custom/public/#{method_name}/#{extra_params}",
+      body: data
+    ]
+
+    request(client, options)
+  end
+
+  @spec invoke_custom_private_method(Tesla.Client.t(), String.t(), String.t(), map()) :: result()
+  def invoke_custom_private_method(client, method_name, extra_params, data) do
```

Comment:
The same as in previous comment.

Reply:
After further discussion, we decided that the `invoke` prefix is more suitable and logical here.